packages/espresso/README.md

```diff
@@ -0,0 +1,115 @@
+# espresso
+
+A new flutter plugin project.
+
+This package provides bindings for Espresso tests of Flutter apps.
+
+## Installation
+
+Add ```android:usesCleartextTraffic="true"``` to your ```<application>``` in your AndroidManifest.xml.
```

Comment:
Might want to clarify this tag is included in the manifest.xml for the testing app only. Production app should remain the same.

Reply:
OK, done